packages/taiko-client/proposer/config.go

```diff
@@ -105,5 +106,10 @@ func NewConfigFromCliContext(c *cli.Context) (*Config, error) {
 			l1ProposerPrivKey,
 			c,
 		),
+		PrivateTxmgrConfigs: pkgFlags.InitTxmgrConfigsFromCli(
+			c.String(flags.L1PrivateTxEndpoint.Name),
```

Comment:
This value might be an empty string, is that ok?

Reply:
It is ok . 
Added `if cfg.PrivateTxmgrConfigs != nil && len(cfg.PrivateTxmgrConfigs.L1RPCURL) > 0` when it was used